packages/web3-eth-abi/src/utils.ts

```diff
@@ -123,12 +123,18 @@ export const mapTypes = (
 	return mappedTypes;
 };
 
+/**
+ * format odd-length bytes to even-length
+ */
+export const formatOddHexstrings = (param: string): string =>
+	param.length % 2 === 1 ? `0x0${param.substring(2)}` : param;
+
```

Comment:
As it's a utility function exported from package, so would be nice to validate the input it's really a hex string and odd lenght.